### ISSUE TITLE
Double-space indentation from line 2 - 6 on _flags.html.erb

### ISSUE THREAD

**Issue by on2onyekachi** [publiclab/plots2#10962]
Hi, this is a [first-timers-only issue](https://code.publiclab.org/#r=all). This means we've worked to make it more legible to folks who either **haven't contributed to our codebase before, or even folks who haven't contributed to open source before**.

If that's you, we're interested in helping you take the first step and can answer questions and help you out as you do. Note that we're especially interested in contributions from people from groups underrepresented in free and open source software!

We know that the process of creating a pull request is the biggest barrier for new contributors. This issue is for you 💝

If you have contributed before, **consider leaving this one for someone new** and looking through our general [help wanted](https://github.com/publiclab/plots2/issues?q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22) issues. Thanks!

### 🤔 What you will need to know.

Nothing. This issue is meant to welcome you to Open Source :) We are happy to walk you through the process.

### 📋 Step by Step

- [ ] 🙋 **Claim this issue**: Claim the issue by commenting. If someone else has claimed it, ask if they've opened a pull request already and if they're stuck -- maybe you can help them solve a problem or move it along!
<details>
  <summary>💡 <i>Learn how to claim</i> 🙋</summary>
  <h3>Claiming an issue</h3>
  <p>Unless the issue is marked as reserved for someone, you can just say "I'd like to try this!" and then you've claimed it - no need to wait for someone to assign it to you. Just be sure you link your pull request (PR) to this issue so we can see where your solution is.</p>
  <p>And open one early if possible - even before you've completed it with additional commits - and others can help you figure out any issues you may face.</p>
</details>

- [ ] 📝 **Update** the file [flags.html.erb](https://github.com/publiclab/plots2/blob/main/app/views/spam2/_flags.html.erb)in the `$plot2 ` repository (press the little pen Icon) and edit the line as shown below.

[See this page](https://code.publiclab.org/#r=all) for some help in taking your first steps!

Below is a "diff" showing in red (and a `-`) which lines to remove, and in green (and a `+`) which lines to add:

```diff
<script>
-$(document).ready(function () {
-	var table = table_main("#flag_table");
-	$("#all").click(function () { //select all button
-		select_all();
- });
-  var pageselect_val = localStorage.getItem('page-select') || '';
-  $("#pageselect").val(pageselect_val);
-	$("#batch-spam").bind('click', function (e) { //batch spam
-		batch_nav("batch_spam");
-	});
-	$("#batch-publish").bind('click', function (e) { //batch publish
-		batch_nav("batch_publish");
-	});
-	$("#batch-ban").bind('click', function (e) { //batch ban
-		batch_nav("batch_ban");
-	});
-	$("#batch-unban").bind('click', function (e) { //batch ban
-		batch_nav("batch_unban");
-	});
-	$("#delete-batch").bind('click', function (e) { // batch delete
-		batch_nav("batch_delete");
-	});
-	$('#pageselect').change(function () {
-		pagination("#pageselect", "/spam2/flags/filter/");
-	});
-	$('#spammed').on('click', function () { //spam filter
-		search_table("spammed", "/spam2/flags/filter/");
-	});
-	$('#unmoderated').on('click', function () { // unmoderated filter
-		search_table("unmoderated", "/spam2/flags/filter/");
-	});
-	$('#page').on('click', function () { // page filter
-		search_table("page", "/spam2/flags/filter/");
-	});
-	$('#note').on('click', function () { // note filter
-		search_table("note", "/spam2/flags/filter/");
-	});
-	$('#reset').on('click', function () { // all filter
-		search_table("all", "/spam2/flags/filter/");
-	});
-});
<script>
+  $(document).ready(function () {
+    var table = table_main("#flag_table");
+    $("#all").click(function () { //select all button
+      select_all();
+    });
+    var pageselect_val = localStorage.getItem('page-select') || '';
+    $("#pageselect").val(pageselect_val);
+    $("#batch-spam").bind('click', function (e) { //batch spam
+      batch_nav("batch_spam");
+    });
+    $("#batch-publish").bind('click', function (e) { //batch publish
+      batch_nav("batch_publish");
+    });
+    $("#batch-ban").bind('click', function (e) { //batch ban
+      batch_nav("batch_ban");
+    });
+    $("#batch-unban").bind('click', function (e) { //batch ban
+      batch_nav("batch_unban");
+    });
+    $("#delete-batch").bind('click', function (e) { // batch delete
+      batch_nav("batch_delete");
+    });
+    $('#pageselect').change(function () {
+      pagination("#pageselect", "/spam2/flags/filter/");
+    });
+    $('#spammed').on('click', function () { //spam filter
+      search_table("spammed", "/spam2/flags/filter/");
+    });
+    $('#unmoderated').on('click', function () { // unmoderated filter
+      search_table("unmoderated", "/spam2/flags/filter/");
+    });
+    $('#page').on('click', function () { // page filter
+      search_table("page", "/spam2/flags/filter/");
+    });
+    $('#note').on('click', function () { // note filter
+      search_table("note", "/spam2/flags/filter/");
+    });
+    $('#reset').on('click', function () { // all filter
+      search_table("all", "/spam2/flags/filter/");
+    });
+  });
```

- [ ] 💾 **Commit** your changes

- [ ] 🔀 **Start a Pull Request**. There are two ways how you can start a pull request:

1. If you are familiar with the terminal or would like to learn it, [here is a great tutorial](https://app.egghead.io/playlists/how-to-contribute-to-an-open-source-project-on-github) on how to send a pull request using the terminal.

2. You can also [edit files directly in your browser](https://help.github.com/articles/editing-files-in-your-repository/) and open a pull request from there.

- [ ] 🏁 **Done** Ask in comments for a review :)

### Please keep us updated

💬⏰ - We encourage contributors to be respectful to the community and provide an update **within a week** of claiming a first-timers-only issue. We're happy to keep it assigned to you as long as you need it if you update us with a request for more time or help, but if we don't see any activity a week after you claim it we may reassign it to give someone else a chance. Thank you in advance!

If this happens to you, don't sweat it! Grab another open issue.

### Is someone else already working on this?

🔗- We encourage contributors to link to the original issue in their pull request so all users can easily see if someone's already started on it.

👥- **If someone seems stuck, offer them some help!** Otherwise, [take a look at some other issues you can help with](https://code.publiclab.org/#r=all). Thanks!

### 🤔❓ Questions?

Leave a comment below, and [reach out in our chatroom](https://publiclab.org/chat)!

**Comment by TildaDares** [comment]
Perfect @on2onyekachi!!!

**Comment by TildaDares** [comment]
Reserved for @Dapper50 for 24 hours!!

**Comment by on2onyekachi** [comment]
> Perfect @on2onyekachi!!!

Thank you, would it be best to break the rest indentations into different issues or create an issue of a lengthy indentation?

**Comment by TildaDares** [comment]
It’d be better to do everything in one PR so that there are no merge conflicts.

**Comment by on2onyekachi** [comment]
Yeah true. Okay,

should I update this to include the rest indentation?

**Comment by TildaDares** [comment]
Yeah you can update this issue.

**Comment by on2onyekachi** [comment]
Okay, done. much better

**Comment by Dapper50** [comment]
@TildaDares please assist review

**Comment by tushmahe** [comment]
Can I work on this issue? @TildaDares

**Comment by tushmahe** [comment]
@TildaDares I have made changes, awaiting your review.

**Comment by Dapper50** [comment]
@tushmahe the issue was reserved for me to work on, which I already have and was awaiting a feedback.

**Comment by tushmahe** [comment]
@Dapper50 Okay! I am extremely sorry for interruption!

**Comment by Dapper50** [comment]
@tushmahe it is fine, I would reach out to you if I need help concerning the project or any other task. Please feel free to do same 🤗

**Comment by Dapper50** [comment]
@tushmahe it is fine, I would reach out to you if I need help concerning the project or any other task. Please feel free to do same 🤗

**Comment by on2onyekachi** [comment]
@tushmahe you can request for FTO at #10153 if you haven't done so.

**Comment by tushmahe** [comment]
@Dapper50 Yeah Sure!!

**Comment by tushmahe** [comment]
@on2onyekachi Okay and Thanks for guiding me !!

**Comment by abdelquodr** [comment]
is this issue done already?

**Comment by on2onyekachi** [comment]
You're welcome @tushmahe . While you wait to be assigned an issue.  

You can go through the project wherever you find an issue or a bug to fix, you can as well create it for yourself and also try to create some for others too.

**Comment by on2onyekachi** [comment]
> is this issue done already?

Yes.

**Comment by abdelquodr** [comment]
ok. Can anyone assign a new issue for me

**Comment by Bhawna-Sharma-9903** [comment]
This issue is yet open, can i work on this?

**Comment by TildaDares** [comment]
Resolved in #10966
